src/Microsoft.DotNet.CodeFormatting/Rules/AttributeNoParenthesesRule.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Linq;
+
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace Microsoft.DotNet.CodeFormatting.Rules
+{
+    [SyntaxRuleOrder(SyntaxRuleOrder.AttributeNoParenthesesRule)]
+    internal sealed class AttributeNoParenthesesRule : ISyntaxFormattingRule
+    {
+        public SyntaxNode Process(SyntaxNode syntaxRoot)
+        {
+            var attributes = syntaxRoot.DescendantNodes()
+                                       .OfType<AttributeSyntax>()
+                                       .Where(a => a.ArgumentList != null &&
+                                                   a.ArgumentList.Arguments.Count == 0 &&
+                                                   (!a.ArgumentList.OpenParenToken.IsMissing || !a.ArgumentList.CloseParenToken.IsMissing));
+
+            return syntaxRoot.ReplaceNodes(attributes, (a, n) => a.WithArgumentList(null));
```

Comment:
Consider the following case:

``` csharp
[Fact(/* Comment */)]
```

This code would strip away that comment.  Was that an intentional decision?  

In general I don't think we should be removing comments in a code formatter.  However I just can't see how such a comment would add value to the system so in this narrow case it seems okay.